src/main/java/com/google/sps/data/GivrUser.java

```diff
@@ -104,6 +115,38 @@ public static void updateUserInDatastore(String identifyingProperty, String iden
     datastore.put(entity);
   }
 
+  public void updateModeratingOrgs() {
+    for (int i = 0; i < this.moderatingOrgs.size(); i++) {
+      OrganizationUpdater organizationUpdater = new OrganizationUpdater(this.moderatingOrgs.get(i));
+      organizationUpdater.updateInvitedModerator(this);
+    }
+  }
+
+  public void setModeratingOrgs() {
+    // There is no need to set moderatingOrgs for maintainers.
```

Comment:
Are you sure? Does this mean that any moderator that happens to be a maintainer never moves off the invited list?

Reply:
That is a great catch! I removed the check for the maintainer status.